spec/features/admin/admin_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'feature_helper'
+
+describe 'ADMIN: AdminUsers' do
+  before { sign_in_as_admin }
+
+  let(:business) { create(:business_complete ) }
+  let(:user) { create(:user, business: business) }
+
+  it 'Creates an admin user' do
+    click_link 'Admin Users'
+    click_on 'New Admin User'
+    fill_in 'Email', with: 'test@example.com'
+    fill_in 'Password', with: 'password'
+    click_on 'Create Admin user'
+    expect(page).to have_content('Admin user was successfully created.')
+  end
+
+  it 'Edits an admin user' do
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Edit Admin User'
+    fill_in 'Password', with: 'password'
+    click_on 'Update Admin user'
+    expect(page).to have_content('Admin user was successfully updated.')
+  end
+
+  @javascript
+  it 'Deletes an Admin User', js: true do
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Delete Admin User'
+    page.evaluate_script('window.confirm = function() { return true; }')
```

Comment:
`page.evaluate_script()` is/looks fairly clunky, bug I tried a few different things, i.e.
`page.driver.browser.switch_to.alert.accept` (error `undefined method 'switch_to'`) and
`page.alert_accept` (error `Capybara::NotSupportedByDriverError: Capybara::Driver::Base#accept_modal`), but the clunky one worked